### DISCUSSION TITLE
Possible to log object?

### DISCUSSION THREAD

**Discussion by Khaled431** [gabime/spdlog#2183] [Q&A] [answered]
I was wondering if it was possible to create a string JIT. I am wanting to print an nlohmanjson object, but would prefer if i could offload the overhead of generating the string to the threadpool.

**Comment by gabime** [comment, marked as answer]
There is no easy way because the fmt formatting is done before sending to the thread pool.

**Reply by Khaled431** [reply]
That's quite unfortunate. Thank you.
